code/upaya/webpack.config.js

```diff
@@ -0,0 +1,35 @@
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './public/js/entry.js',
+
+  output: {
+    filename: 'bundle.js',
+    path: __dirname + '/public/js'
+  },
+
+  module: {
```

Comment:
Define a sourcemap for sanity in debugging?

https://github.com/department-of-veterans-affairs/vets-website/blob/master/webpack.config.js#L13